macros/tuning_parameters.mac

```diff
@@ -11,18 +11,8 @@
 # Set top veto PMT spacing in cm.
 /WCSim/tuning/tvspacing 100
 
-
-# Paramater to set photocathode model
-# Set to "0" to use default model, "1" or "2" to use new models
-# "1" is copied from geant4.11, which is based on https://ieeexplore.ieee.org/document/9875513
-# Model the alloy as a thin layer with real refractive index, then calculate reflection and transmission probability
-# Cannot handle total internal reflection when n1<n2
-# "2" is implemented based on https://arxiv.org/abs/physics/0408075v1
-# Model the alloy as a thin layer with real and imaginary refractive indices, then calculate absorption, reflection and transmission probability
-/WCSim/tuning/pmtsurftype 0
-# Choice of photocathode optical parameters.
-# 0 = (Seems to be) SK. 1 = KCsCb, 2 = RbCsCb [1 and 2 from from pmtsurftype Model2's paper]
-/WCSim/tuning/cathodepara 0
+# Input file to set photocathode parameters
+#/WCSim/tuning/PMTCathodePara data/CathodeParameters_KCsCb.txt
```

Comment:
Please setup a new txt file that corresponds to the default, that is the file path that is commented out here (the WCSim convention is that if there is only one example config line given, it corresponds to the default option)

(I think that is approximately CathodeParameters_SK.txt with `photocathode_model_choice` set to 0)